views/vm/create.erb

```diff
@@ -66,6 +66,15 @@
                   }
                 ) %>
               </div>
+              <div class="flex items-center">
+                <input
+                  id="enable-ip4"
+                  name="enable-ip4"
+                  type="checkbox"
+                  class="h-4 w-4 rounded border-gray-300 text-orange-600 focus:ring-orange-600"
+                >
+                <label for="enable-ip4" class="ml-2 block text-sm text-gray-900">Enable Public IPv4</label>
+              </div>
```

Comment:
We have checkbox component. Also you can show subject select and ipv4 checkbox side by side on large screens.
```suggestion
              <div class="sm:col-span-6">
                    <%== render(
                      "components/form/checkbox",
                      locals: {
                        name: "enable-ip4",
                        label: "Enable Public IPv4",
                        value: "1"
                      }
                    ) %>
              </div>
```

Also change `class="col-span-full"` to `class="sm:col-span-6">` for subnet select too.